### ISSUE TITLE
Can't get value 

### ISSUE THREAD

**Issue by Lat9898** [stas-sl/esphome-sound-level-meter#10]
Hej I have an issue that it is only working with wrong channel. f.eks. when i set L/R to GND so the channel should be left but it is only working with right channel which is teoritically wrong. and the lowest i get is 75 dBA in a quiet room which is totally wrong. 
Do you have any tips that i can do? 
I am using INMP441 mic and ESP32 devkitc v4.
Have you had also those issue when u working with Ivan's code.

**Comment by stas-sl** [comment]
Yes, there were several reports about L/R channels being swapped ([1](https://github.com/espressif/esp-idf/issues/6625), [2](https://github.com/ikostoski/esp32-i2s-slm/blob/b643efa1beae51b84d4ad9b2bfa35d439c81e9b6/esp32-i2s-slm.ino#L245-L253)) Although, it should be already fixed, seems like it still occurs (in some versions of esp-idf?)

There could be various reasons why you are getting wrong values. Unfortunately just final value of 75 dBA doesn't tell much. It requires more debugging e.g. monitoring/printing raw absolute values (hex or decimal) like it was discussed [here](https://github.com/ikostoski/esp32-i2s-slm/issues/17).
